roles/redpanda_broker/defaults/main.yml

```diff
@@ -9,13 +9,18 @@ redpanda_kafka_port: 9092
 redpanda_rpc_port: 33145
 
 redpanda_use_staging_repo: false
-redpanda_repo_debian: "{{ 'https://packages.vectorized.io/sMIXnoa7DK12JW4A/redpanda/cfg/setup/bash.deb.sh' if not redpanda_use_staging_repo|bool else 'https://dl.redpanda.com/E4xN1tVe3Xy60GTx/redpanda-unstable/setup.deb.sh' }}"
-redpanda_repo_redhat: "{{ 'https://dl.redpanda.com/nzc4ZYQK3WRGd9sy/redpanda/cfg/setup/bash.rpm.sh' if not redpanda_use_staging_repo|bool else 'https://dl.redpanda.com/E4xN1tVe3Xy60GTx/redpanda-unstable/setup.rpm.sh' }}"
+redpanda_base_url: "https://dl.redpanda.com"
+
+redpanda_repo_debian: "{{ 'https://packages.vectorized.io/sMIXnoa7DK12JW4A/redpanda/cfg/setup/bash.deb.sh' if not redpanda_use_staging_repo|bool else '{{ redpanda_base_url }}/E4xN1tVe3Xy60GTx/redpanda-unstable/setup.deb.sh' }}"
+redpanda_repo_redhat: "{{ '{{ redpanda_base_url }}/nzc4ZYQK3WRGd9sy/redpanda/cfg/setup/bash.rpm.sh' if not redpanda_use_staging_repo|bool else '{{ redpanda_base_url }}/E4xN1tVe3Xy60GTx/redpanda-unstable/setup.rpm.sh' }}"
```

Comment:
[question]

1. Why Debian default to `packages.vectorized` and redhat uses `dl.redpanda` ? I thought we were migrating away from packages.vectorized.
2. Why using the unstable version?

Reply:
Mostly because that's what they were before and I haven't looked into what the correct answer would be.

Reply:
Can we ask devprod and update the vectorized one please? I fear that they might drop that in favor of cloudsmith (dl.redpanda).

Maybe something for another PR, lmk if that's the case and I'll approve.

Reply:
Asked https://redpandadata.slack.com/archives/C02LZGSS66M/p1685051234352679

Reply:
Oh right I'm not using these anymore, deleted